src/app/shared/form/builder/ds-dynamic-form-ui/models/autocomplete/dynamic-autocomplete.component.ts

```diff
@@ -0,0 +1,296 @@
+import {ChangeDetectorRef, Component, EventEmitter, Input, OnInit, Output, TemplateRef, ViewChild} from '@angular/core';
+import {FormGroup} from '@angular/forms';
+import {DynamicTagModel} from '../tag/dynamic-tag.model';
+import {NgbTypeahead, NgbTypeaheadSelectItemEvent} from '@ng-bootstrap/ng-bootstrap';
+import {Observable, of as observableOf} from 'rxjs';
+import {PageInfo} from '../../../../../../core/shared/page-info.model';
+import {VocabularyService} from '../../../../../../core/submission/vocabularies/vocabulary.service';
+import {DynamicFormLayoutService, DynamicFormValidationService} from '@ng-dynamic-forms/core';
+import {catchError, debounceTime, distinctUntilChanged, map, merge, switchMap, tap} from 'rxjs/operators';
+import {buildPaginatedList, PaginatedList} from '../../../../../../core/data/paginated-list.model';
```

Comment:
with so many ../ .. it is like ASCII art

╱╱▏╱▏╱▏╱╱╱╱╱╱▏╱╱╱╱╱▏
▉╱▉╱▉╱▏▉▉▉▉▉╱┈▉▉▉▉╱
▉╱▉╱▉╱▏┈┈▉╱▏┈┈▉╱╱╱╱▏
▉╱▉╱▉╱▏┈┈▉╱▏┈┈▉▉▉▉╱
▉▉▉▉▉╱┈┈┈▉╱┈┈┈▉╱

Reply:
Yes, but imports like that are through whole dspace-angular. check: https://github.com/dataquest-dev/dspace-angular/blob/dtq-dev/src/app/item-page/edit-item-page/item-status/item-status.component.ts